modulefiles/core/cmakemodules/cmakemodules.lua

```diff
@@ -0,0 +1,21 @@
+help([[
+]])
+
+local pkgName    = myModuleName()
+local pkgVersion = myModuleVersion()
+local pkgNameVer = myModuleFullName()
+
+conflict(pkgName)
+
+local opt = os.getenv("HPC_OPT") or os.getenv("OPT") or "/opt/modules"
+
+local base = pathJoin(opt,"core",pkgName,pkgVersion)
+
+prepend_path("CMAKE_MODULE_PATH", pathJoin(base,"Modules"), ";")
```

Comment:
This is not an appropriate variable to set.  The use of this variable is in cmake and cmake does not look at this environment variable.  To use this variable with cmake, one has to pass it as an command line argument, like so:
```
cmake -DCMAKE_MODULE_PATH=/path/to/append/to/default/CMAKE_MODULE_PATH
```

I suggest using the environment variable `CMAKEMODULES_ROOT` defined in this modulefile wherever necessary.

Reply:
This setting was removed in f9f91faba8371301c2248bf53303fd6783579cba